certify-service/src/main/java/io/mosip/certify/proof/JwtProofKeyManager.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package io.mosip.certify.proof;
+
+import com.nimbusds.jose.JWSHeader;
+import com.nimbusds.jose.jwk.JWK;
+
+import java.util.Optional;
+// Should this method be an abstract class instead of an interface for managing holder's key in JWK format?
+/**
+ * {@link JwtProofKeyManager} helps in managing the holder's key.
+ */
+public interface JwtProofKeyManager {
```

Comment:
Lets remove this whole interface and implementations and replace it with didkit or waltid libary based resolving.

Reply:
Lets create a JIRA for next release.